pkg/ingress/model_build_target_group.go

```diff
@@ -74,29 +74,38 @@ func (t *defaultModelBuildTask) buildTargetGroupBindingSpec(ctx context.Context,
 	}
 }
 
-func (t *defaultModelBuildTask) buildTargetGroupBindingNetworking(ctx context.Context) *elbv2model.TargetGroupBindingNetworking {
+func (t *defaultModelBuildTask) buildTargetGroupBindingNetworking(ctx context.Context, targetGroupPort int64, healthCheckPort intstr.IntOrString) *elbv2model.TargetGroupBindingNetworking {
```

Comment:
Lets make the model builder revert to prior behavior if `disable-restricted-sg-rules` is set to true so that users can get existing behavior in case.

Reply:
Changed.